.gitignore

```diff
@@ -2,9 +2,11 @@
 .idea/
 local/*
 !local/README
-/data/
+/data/*
 !data/*.sh
 !data/preprocess.py
+!data/preprocess/
```

Comment:
move all code there; no package. I suggest download script calls preprocess/preprocess-default.py or preprocess/preprocess-<dataset>.py. All utility functions go to preprocess/util.py